build_tools/benchmarks/common/benchmark_thresholds.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""A list of benchmarks and their similarity thresholds."""
+
+import re
+
+from dataclasses import dataclass
+from enum import Enum
+
+
+class ThresholdUnit(Enum):
+  PERCENTAGE = 0
+  VALUE = 1
```

Comment:
Arbitrary units, determined by the specific benchmark? Or if always milliseconds, `VALUE_MS`?

Reply:
Good point. Better to be explicit.